man/rgbgfx.1

```diff
@@ -164,6 +165,37 @@ for a list of formats and their descriptions.
 .It Fl d Ar depth , Fl \-depth Ar depth
 Set the bit depth of the output tile data, in bits per pixel (bpp), either 1 or 2 (the default).
 This changes how tile data is output, and the maximum number of colors per palette (2 and 4 respectively).
+.It Fl i Ar input_tiles , Fl \-input-tileset Ar input_tiles
+Read tiles that will be used to convert this image, and that will always be given the first IDs.
+.Ar input_tiles
+must contain tile data in
+.Dq raw
+format, as generated through the
```

Comment:
I don't think we describe Game Boy graphics as "raw" anywhere; if anything they're the processed form, PNG is raw. We do refer to "2bpp tile data" in a few places.
```suggestion
must contain 2bpp tile data, as could be previously generated with the
```